<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KundeNeu</class>
 <widget class="QWidget" name="KundeNeu">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Neuer Kunde</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>54</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Stra√üe:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditOrt">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>161</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPLZ">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>161</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditName">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>161</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>54</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>PLZ:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditVorname">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>161</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>54</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Ort:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonBeenden">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>210</y>
     <width>95</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Beenden</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>54</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Telefon:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>54</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>54</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Vorname:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditTelefon">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>160</y>
     <width>161</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonOK">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>95</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditStrasse">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>161</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEditName</tabstop>
  <tabstop>lineEditVorname</tabstop>
  <tabstop>lineEditStrasse</tabstop>
  <tabstop>lineEditPLZ</tabstop>
  <tabstop>lineEditOrt</tabstop>
  <tabstop>lineEditTelefon</tabstop>
  <tabstop>buttonOK</tabstop>
  <tabstop>buttonBeenden</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
